core/src/main/java/com/novoda/noplayer/PlayerBuilder.java

```diff
@@ -4,6 +4,8 @@
 import android.os.Handler;
 import android.os.Looper;
 
+import com.google.android.exoplayer2.source.MediaSource;
+import com.google.android.exoplayer2.upstream.DataSource;
```

Comment:
❌ The `no-player` specific classes cannot know about the internals of `exo-player`. `exo-player` classes can only be exposed in either the `com.novoda.noplayer.external.exoplayer` or `com.novoda.noplayer.internal` packages.